core/io/fixtures/classes.rb

```diff
@@ -71,7 +71,7 @@ def self.lines_space_separator_limit
   end
 
   def self.lines_r_separator
-    [ "Voici la ligne une.\nQui \303\250 la linea due.\n\n\n" \
+    [ "Voici la ligne une.\nQui \303\250 la linea due.\n\n\n" << \
```

Comment:
I think we could just have the entire string as a single literal.
It's a bit long, but then it's clearer it's just one String and there is no concatenation involved.